eng/pipelines/agent-pool-migration.yml

```diff
@@ -0,0 +1,138 @@
+trigger: none
+pr: none
+
+parameters:
+- name: Repos
+  type: object
+  default:
+    # The exclude/include paths are valid for ubuntu agents. If use windows, please follow the format like "^.\\sdk\\" 
+    - RepoOwner: Azure
+      RepoName: azure-sdk-tools
+      ExcludePaths: ''
+      IncludePaths: ''
+      Branch: 'main'
+    - RepoOwner: Azure
+      RepoName: azure-sdk
+      ExcludePaths: "^./eng/common/*"
+      IncludePaths: ''
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-android
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/resources/session-records/"
+      IncludePaths: "^./sdk/.*-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-c
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-cpp
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-go
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-ios
+      ExcludePaths: "^./sdk/|^./eng/common/"
+      IncludePaths: "^./sdk/.*-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-java
+      ExcludePaths: "^./sdk/|^./eng/common/|/resources/session-records/"
+      IncludePaths: "^./sdk/.*-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-js
+      ExcludePaths: "^./sdk/|^./eng/common/|/recordings/node/"
+      IncludePaths: "^./sdk/.*-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-net
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/SessionRecords/"
+      IncludePaths: "^./sdk/.*-matrix.*.json$|^./sdk/.*/.*yml$"
+    - RepoOwner: Azure
+      RepoName: azure-sdk-for-python
+      ExcludePaths: "^./sdk/|^./eng/common/|/tests/recordings/"
+      IncludePaths: "^./sdk/.*-matrix.*.json$|^./sdk/.*/.*yml$"
+# Default setting is an example of the map looks like:
+- name: MigrationMap
+  type: object
+  default: 
+    - MigrateFrom: azsdk-pool-mms-win-2022-general
+      MigrateTo: azsdk-pool-mms-win-2022-general
+    - MigrateFrom: windows-2022
+      MigrateTo: windows-2022
+    - MigrateFrom: windows2022
+      MigrateTo: windows2022
+    - MigrateFrom: MMS2019
+      MigrateTo: MMS2022
+- name: ShortForMigrateFrom
+  type: string
+  default: ''
+- name: ShortForMigrateTo
+  type: string
+  default: ''
+# The value here is github alias: e.g. sima-zhu
+- name: GithubAssigneeAlias
+  type: string
+  default: ''
+- name: WhatIfPreference
+  type: boolean
+  default: false
+
+variables:
+  - template: /eng/pipelines/templates/variables/globals.yml
+jobs:
+  - job: Migration
+    pool:
+      name: azsdk-pool-mms-win-2022-general
```

Comment:
In the default `Repos` parameter you used path format that works on Ubuntu, not on Windows. But by default you use Windows pool. Shouldn't this pool be Ubuntu instead, so the `Repos` paths work?

Reply:
I should make changes to make comments. Windows pool is compatible with both of the styles. We used ubuntu pool before which has limitation on the path.

The reason of switching to win pool is I found ubuntu pool doesn't read the files under folder like `.azure-pipeline`